.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
This file shouldn't be part of the PR, the entire `.idea` folder should not be committed in this PR, please remove it